lib/tests/streamlit/watcher/event_based_path_watcher_test.py

```diff
@@ -42,14 +40,13 @@ def tearDown(self):
 
         self.observer_class_patcher.stop()
         self.util_patcher.stop()
-        self.os_patcher.stop()
 
     def test_file_watch_and_callback(self):
         """Test that when a file is modified, the callback is called."""
         cb = mock.Mock()
 
-        self.os.stat = lambda x: FakeStat(101)
-        self.mock_util.calc_md5_with_blocking_retries = lambda x: "1"
+        self.mock_util.path_modification_time = lambda *args: 101
```

Comment:
nit-pick: real `path_modification_time` return type is `float`, so here we can return `101.0` and `102.0` just for "correctness"

Reply:
👍 done